server/routes.js

```diff
@@ -35,6 +35,7 @@ function configureRoutes(app) {
   app.post('/pa/get-sensor-clients', pa.validateGetSensorClients, googleHelpers.paAuthorize, pa.handleGetSensorClients)
   app.post('/pa/sensor-twilio-number', pa.validateSensorPhoneNumber, googleHelpers.paAuthorize, pa.handleSensorPhoneNumber)
   app.post('/pa/message-clients', pa.validateMessageClients, googleHelpers.paAuthorize, pa.handleMessageClients)
+  app.post('/pa/check-database-connection', pa.validateCheckDatabaseConnection, googleHelpers.paAuthorize, pa.handleCheckDatabaseConnection)
```

Comment:
Will `googleHelpers.paAuthorize` work with a `GET` request?

I feel like that would be more appropriate than a `POST` in this case since it's getting the information of whether it's online or not.

Reply:
Good question! I originally tried a GET request but I realized `googleHelpers.paAuthorize` looks for the `googleIdToken` in `req.body` and if I made a GET request in PA, data can only get passed via parameters (as opposed to a POST request, where you can pass data via the request body). So because of that I chose to use a POST request.

Reply:
Gotcha. I do think that we'll eventually want `GET` calls to work from PA, when we start moving the dashboard over there. So it's not something that needs to be changed now, but maybe @micahdbak can think about what changes that might entail.

Reply:
The main problem with authenticating a GET request is that the googleIdToken is far too long to be contained in the query string of a request... I think we could look into setting Authorization headers for the request? Though I have no experience with that...

Reply:
Oh! OK. I wonder how other people do it.